python-avd/pyavd/_eos_designs/structured_config/network_services/static_routes.py

```diff
@@ -55,30 +51,21 @@ def static_routes(self: AvdStructuredConfigNetworkServices) -> list[dict] | None
                             # Only create static routes for VARP entries with masks
                             continue
 
-                        static_route = {
-                            "destination_address_prefix": str(ipaddress.ip_network(virtual_router_address, strict=False)),
-                            "vrf": vrf.name,
-                            "name": "VARP",
-                            "interface": f"Vlan{svi.id}",
-                        }
+                        static_route_dict = EosCliConfigGen.StaticRoutesItem(
+                            destination_address_prefix=str(ipaddress.ip_network(virtual_router_address, strict=False)),
+                            vrf=vrf.name,
+                            name="VARP",
+                            interface=f"Vlan{svi.id}",
+                        )
 
-                        # Ignore duplicate items in case of duplicate VRF definitions across multiple tenants.
-                        if static_route not in static_routes:
-                            static_routes.append(static_route)
+                        self.structured_config.static_routes.append(static_route_dict)
 
         for _internet_exit_policy, connections in self._filtered_internet_exit_policies_and_connections:
             for connection in connections:
                 if connection["type"] == "tunnel":
-                    static_route = {
-                        "destination_address_prefix": f"{connection['tunnel_destination_ip']}/32",
-                        "name": f"IE-ZSCALER-{connection['suffix']}",
-                        "gateway": connection["next_hop"],
-                    }
-                    # Ignore duplicate items in case of multiple connections generating the same route
-                    if static_route not in static_routes:
-                        static_routes.append(static_route)
-
-        if static_routes:
-            return static_routes
-
-        return None
+                    static_route = EosCliConfigGen.StaticRoutesItem(
+                        destination_address_prefix=f"{connection['tunnel_destination_ip']}/32",
+                        name=f"IE-ZSCALER-{connection['suffix']}",
+                        gateway=connection["next_hop"],
+                    )
+                    self.structured_config.static_routes.append(static_route)
```

Comment:
```suggestion
                    # Ignore duplicate items in case of multiple connections generating the same route
                    if static_route not in static_routes:
                        self.structured_config.static_routes.append(static_route)
```